Source/WebCore/platform/graphics/cocoa/MediaPlayerPrivateWebM.mm

```diff
@@ -1131,6 +1135,8 @@ static bool isCopyDisplayedPixelBufferAvailable()
 {
     ALWAYS_LOG(LOGIDENTIFIER, "data length = ", buffer.size());
 
+    setNetworkState(MediaPlayer::NetworkState::Loading);
```

Comment:
Why is it necessary?

Reply:
to be more spec compliant.
we are here if we have started downloading content.
https://html.spec.whatwg.org/multipage/media.html#concept-media-load-algorithm

it will also let the media element queue a loadstart event.

I'll amend the commit message of a fly-by fix.